frontend/src/pages/PropertyCreation.jsx

```
@@ -0,0 +1,710 @@
+          <form onSubmit={handleSubmit} noValidate>
+            {/* Basic Information */}
+            <div className="mb-8">
+              <h2 className="text-xl font-semibold mb-4 text-gray-700 border-b pb-2">Basic Information</h2>
+              <div className="grid grid-cols-1 md:grid-cols-2 gap-6">
+                <div>
+                  <label className="block text-gray-700 mb-2">
+                    Property Name*
+                    {hasError('name') && (
+                      <span className="text-red-500 ml-1 text-sm">{errors.name}</span>
+                    )}
+                  </label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)